Guide/src/reference/architecture/openhcl_servicing.md

```diff
@@ -0,0 +1,258 @@
+# OpenHCL Servicing
```

Comment:
Clarify that there are things that happen in and outside of blackout. "Guest VTL0 VP Blackout" is the important performance metric: it is the time that VTL0 Guest VPs are unable to run. (I put quotes around that, but pick a good way to represent that term).

Save/Restore happen in blackout, for example.